OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.cpp

```diff
@@ -160,14 +79,38 @@ calcDerivative(double activation, double excitation) const
 {
```

Comment:
Good point Chris. Sorry Tom @tkuchida can you file an issue so this doesn't get forgotten.